.github/workflows/lint_python.yml

```diff
@@ -0,0 +1,24 @@
+name: lint_python
+on: [pull_request, push]
+jobs:
+  lint_python:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - run: pip install --upgrade pip wheel
+      - run: pip install bandit black codespell flake8 isort mypy pytest pyupgrade safety
+      - run: bandit --recursive --skip B101 . || true  # B101 is assert statements
+      - run: black --check . || true
+      - run: codespell || true  # --ignore-words-list="" --skip="*.css,*.js,*.lock"
+      - run: flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+      - run: flake8 . --count --exit-zero --max-complexity=10 --max-line-length=88 --show-source --statistics
+      - run: isort --check-only --profile black . || true
```

Comment:
I would remove all the pytest related code here, as we are already doing an actual pytest in CI. See https://github.com/isl-org/Open3D/blob/44e9c4713bd86a83b6fa7141faaeb28558c41180/util/ci_utils.sh#L394 and https://github.com/isl-org/Open3D/runs/3237531611#step:8:1

Reply:
pytest removed